precompiles/bank/bank.go

```diff
@@ -116,6 +120,11 @@ func (p Precompile) Run(evm *vm.EVM, caller common.Address, input []byte) (bz []
 			return nil, err
 		}
 		return p.send(ctx, method, args)
+	case SendNativeMethod:
+		if err := p.validateCaller(ctx, caller); err != nil {
```

Comment:
`validateCaller` would validate that the `caller` is a whitelisted contract (i.e. our ERC20 wrappers), whereas the use case for sending to native address is for users to call directly, so let's not call it here, and change the function signature to:
```
function sendNative(
        string memory toNativeAddress,
        string memory denom,
        uint256 amount
    ) external returns (bool success);
```
The `from` address is just `caller`